boards/thingy91/doc.txt

```diff
@@ -0,0 +1,25 @@
+/**
+ * @defgroup    boards_thingy91 Thingy:91
+ * @ingroup     boards
+ * @brief       Support for the Thingy:91 board.
+ *
+ * ### General information
+ *
+ * Some general information about the board
+ *
+ * ### Pinout
+ *
+ * A detailed description of the board pinout (if available)
+ * 
+ * ### Flash the board
+ * Used extra software: https://eclipse-embed-cdt.github.io/debug/jlink/install/ 
+ * 
+ * Some notes about how to flash the board
+ * BOARD=thingy91 make flash term
+ */
+
+compilation to run
```

Comment:
Leftover?
btw the board can also be flashed (and `stdio_rtt`also works with) OpenOCD